pkg/jenkinsfile/pipeline.go

```diff
@@ -571,9 +571,11 @@ func (a *CreateJenkinsfileArguments) GenerateJenkinsfile(resolver ImportFileReso
 	}
 	outFile := a.OutputFile
 	outDir, _ := filepath.Split(outFile)
-	err = os.MkdirAll(outDir, util.DefaultWritePermissions)
-	if err != nil {
-		return errors.Wrapf(err, "failed to make directory %s", outDir)
+	if outDir != "" {
+		err = os.MkdirAll(outDir, util.DefaultWritePermissions)
+		if err != nil {
+			return errors.Wrapf(err, "failed to make directory '%s' when creating Jenkinsfile '%s'", outDir, outFile)
```

Comment:
Can you remove the single quotes around the variables to be consistent with the rest of Jenkins X please